gvfs.md

```diff
@@ -160,18 +161,80 @@ The following fields are all attributes within the main "data" object for this f
+\- image_large | Optional | A url to a an image with a resolution larger than 800px x 400px for the purposes of displaying a full-size image in an end-user application
+\- propulsion_type | Yes | An enumerable describing the propulsion type of the vehicle. <br /><br />Current valid values (in CAPS) are:<br /><ul><li>HUMAN _(Pedal or foot propulsion)_</li><li>ELECTRIC_ASSIST _(Provides power only alongside human propulsion)_</li><li>ELECTRIC _(Contains throttle mode with a battery-powered motor)_</li><li>COMBUSTION _(Contains throttle mode with a gas engine-powered motor)_</li></ul> This field was copied from [City of Los Angeles Mobility Data Specification](https://github.com/CityOfLosAngeles/mobility-data-specification/blob/73995a151f0a1d67aab3d617a4693f8f81967936/provider/README.md#propulsion-types)
+\- cruising_speed | Optional | The recommended top cruising speed that the vehicle can travel at in a safe manner in kilometers per hour
+\- range_with_full_energy_potential | Optional | The furthest distance in kilometers that the vehicle can travel when it has the maximum amount of energy potential (for example a full battery or full tank of gas)
+\- vehicle_weight | Optional | The weight in kilograms of the vehicle
+\- power_output | Optional | The maximum power output in watts that can be generated with the vehicle's motor during a safe and normal acceleration of the vehicle
+\- passenger_capacity | Optional | The amount of people than can travel at once on this vehicle
+\- enclosed | Optional | 1/0 boolean - whether the vehicle contains an enclosed environment that isolates the passenger(s) from the outside elements
+
+Example:
+
+```json
+{
+  "last_updated": 1434054678,
+  "ttl": 12345,
+  "data": {
+    "system_id": "inboard",
+    "language": "en",
+    "name": "Inboard",
+    "short_name": "ib",
+    "url": "https://www.inboardtechnology.com/",
+    "timezone": "America/Los_Angeles",
+    "vehicle_types": [
+      {
+        "vehicle_type_id": "m1",
+        "name": "M1 Electric Skateboard",
+        "description": "Experience the most advanced e-board deck ever developed. A custom designed composite deck that combines an inverted 3D Poplar wood core with full sandwich PU sidewalls and wrapped in the same fiberglass top sheet found in premium snowboards and skis. Integrated electronics, reinforced truck mounting points, and resilient TPU nose and tail bumpers at the rails yield a remarkably light and durable platform optimized for a more responsive ride.",
+        "vehicle_url": "https://www.inboardtechnology.com/products/m1-electric-skateboard",
+        "image_small": "http://cdn.shopify.com/s/files/1/1136/4406/files/Skateboard_x150.png?11016692238788873189",
+        "image_large": "http://cdn.shopify.com/s/files/1/1136/4406/products/m1_pdp_fullBleed_6_9e6614b5-c3e4-4dd1-b18b-fd51451d0c17_1600x.jpg?v=1535665708",
+        "propulsion_type": "ELECTRIC",
```

Comment:
maybe add a field voor vehicle_type as well?

bike/scooter/car/motor

Reply:
I intentionally avoided this kind of field because there are already so many kinds of micromobility vehicles being made that may not be classifiable in a commonly understood way. In the example json in this PR, I included an electric skateboard which already falls outside the proposed types you have listed. This kind of classification of the vehicle type I believe is best left for interpretation by the gvfs-consuming application. In OpenTripPlanner I forsee adding updaters that would parse each vehicle type and assign it a various operational characteristics that would determine how the vehicle would be able to travel through the street network in the routing algorithms.

Reply:
I think it doesn't hurt to add a vehicle_type for all mainstream modes. If new mainstream modes appear they can be added as well to the specification without breaking previous api's. We could also specify 'other' for all other modes that should be derived.